velox/type/UnscaledShortDecimal.h

```diff
@@ -69,6 +93,8 @@ struct UnscaledShortDecimal {
 
  private:
   int64_t unscaledValue_;
+  static constexpr int64_t kMin = -1'000'000'000'000'000'000 + 1;
```

Comment:
nit: put static constexpr ahead of non-static members?

Reply:
done